user_tools/src/spark_rapids_tools/cmdli/argprocessor.py

```diff
@@ -384,10 +384,13 @@ def process_tools_config(self) -> None:
             try:
                 self.p_args['toolArgs']['toolsConfig'] = ToolsConfig.load_from_file(self.tools_config_path)
             except ValidationError as ve:
+                # If required, we can dump the expected specification by appending
+                # 'ToolsConfig.get_schema()' to the error message
                 raise PydanticCustomError(
-                    'invalid_argument',
+                    'invalid_config',
                     f'Tools config file path {self.tools_config_path} could not be loaded. '
-                    'It is expected to be a valid YAML file.\n  Error:') from ve
+                    f'It is expected to be a valid configuration YAML file.'
```

Comment:
nit: `f'It is expected to be a valid configuration YAML file.'` does not need formatting.

Reply:
Fixed!